### PR TITLE
Milestone 1: Book an Appointment Front End | Setup Repository

### PR DESCRIPTION
**Pull request by samiullah997**
## Milestone 1: Book an Appointment Front End | Setup Repository
In this milestone:
- Create a Repository for the `Book an Appointment Front` End in React.
- Added `Tailwindcss` for styling.
- Create a `SignUp` form to register a new user.
- Create a `login` form to log in the register the user.
- Create a `Home` page to show all lists of bikes.
- Fix all linter errors.

### CODE REVIEW

File: src/components/authentication/LogIn.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import { SigninDetailsApi } from '../../redux/reducer/user';
+
+const LogIn = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const [email, setEmail] = useState('');
+  const [password, setPassowrd] = useState('');
+  const state = useSelector((state) => state.UserReducer);
+  const { loggedIn } = state;
+  // const [signedInSuccess, setSignedInSuccess] = useState(loggedIn);
```
Comment (Alfred-KInyua) [inline src/components/authentication/LogIn.js:13]: Please remove commented code as it clutters your work.

Comment (samiullah997) [reply src/components/authentication/LogIn.js:13]: Thank @Alfred-KInyua  to mention the commented code.

**Review by Alfred-KInyua — CHANGES_REQUESTED** [review]
Hi @samiullah997 👋🏿 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfred-KInyua — APPROVED** [review]
Hi @samiullah997 ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 👍🏿 
Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding 👏🏿 👏🏿 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by samiullah997** [comment]
Thanks, @Alfred-KInyua. to give me good feedback. Happy Coding :wave:

**Pull request merged**